tdvt/tdvt/tdvt_core.py

```diff
@@ -32,13 +32,14 @@
 
 
 class ConnectorsTest(object):
-    def __init__(self, conn_test_name, conn_test_file):
+    def __init__(self, conn_test_name, conn_test_file, conn_test_password_file):
```

Comment:
Can you verify this will work without a password file specified? Similar comment to above, probably need 
```suggestion
    def __init__(self, conn_test_name, conn_test_file, conn_test_password_file=None):
```

Reply:
Yes i had tested this case before sending out the PR. It works correctly with and without the password file